# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../zopeinterface/gcc41.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ZopeInterface-3.0.1/Dependencies/zope.interface-ZopeInterface-3.0.1/zope.interface/_zope_interface_coptimizations.c.vanilla	2006-11-18 17:35:08.000000000 +0100
+++ ZopeInterface-3.0.1/Dependencies/zope.interface-ZopeInterface-3.0.1/zope.interface/_zope_interface_coptimizations.c	2006-11-18 17:35:13.000000000 +0100
@@ -336,7 +336,7 @@
 	{NULL,		NULL}		/* sentinel */
 };
 
-static PyTypeObject SpecType = {
+PyTypeObject SpecType = {
 	PyObject_HEAD_INIT(NULL)
 	/* ob_size           */ 0,
 	/* tp_name           */ "_interface_coptimizations."
